### DISCUSSION TITLE
giveaway image

### DISCUSSION THREAD

**Discussion by patrikpereira99** [sandarutharuneth/ivongiveaways#26] [Q&A] [answered]
any possibility to add an embedImage on giveaway after use start? like put an image from the game or thing u are giving..

**Comment by sandarutharuneth** [comment]
You can easily add that on start.js
I've added a embed to add extra details of the embed if needed. In there you can add a `.setImage()` function `( line 151 - 165 )` and add it to slash commands as an option. Update as follow!

**Update the slash commands options. ( Add this right after description option )**
```js
{
      name: 'image',
      description: 'Add and image to the extra details embed',
      type: ApplicationCommandOptionType.String,
      required: false
},
```
**Import the image component**
```js
const image = interaction.options.getString('image');
```

**Update the code on line 151 - 165** 
```js
if (invite) {
      let des = new Discord.EmbedBuilder()
        .setAuthor({ name: `Extra Details` })
        .setDescription(`${invite}`)
        .setImage(`${image}`)
        .setColor("#2F3136");

      const row = new ActionRowBuilder()
    .addComponents(
        new ButtonBuilder()
        .setLabel('Twitter')
        .setStyle(ButtonStyle.Link)
        .setURL(`https://twitter.com/ivongiveaways`))

      giveawayChannel.send({ embeds: [des], components: [row] });
    }
```
Now you have a optional feature to add images to the giveaway!

**Reply by patrikpereira99** [reply]
look at my start.js

`https://github.com/SrPalha/teste/blob/main/start.js`

no like i did all right but don't work as i want like this screenshot
<img width="387" alt="Screen Shot 2023-10-06 at 23 47 57" src="https://github.com/sandarutharuneth/ivongiveaways/assets/6546195/71c7a548-b4ae-47cb-9167-279e3938f9e1">

**Reply by sandarutharuneth** [reply, marked as answer]
Then you have to customize it yourself for your liking.
